### PR TITLE
Welsh amends

### PR DESCRIPTION
**Pull request by davidrapson**
Makes amends to welsh language text based on review.

### CODE REVIEW

File: controllers/apply/form-router-next/views/eligibility.njk [hunk 1]
```diff
@@ -34,6 +27,13 @@
                     </p>
                 </div>
             {% else %}
+                {% set prefix = __('global.misc.stepProgress', currentStepNumber, totalSteps) %}
```
Comment (davidrapson) [inline controllers/apply/form-router-next/views/eligibility.njk]: Realised whilst testing this that this header doesn't make sense on the success / ineligible screens to switched this around

File: controllers/apply/awards-for-all/fields/your-idea-priorities.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ module.exports = function fieldYourIdeaPriorities(locale) {
                 ariannu, dywedwch wrthym sut bydd eich prosiect yn
                 <strong>bodloni o leiaf un o’r rhain</strong>:
             </p>
-            <li>
+            <ol>
```
Comment (mattandrews) [inline controllers/apply/awards-for-all/fields/your-idea-priorities.js]: Nice catch 👍

File: controllers/apply/awards-for-all/terms.js [hunk 3]
```diff
@@ -144,7 +144,7 @@ module.exports = {
             ]
         },
         {
-            title: `You 2.	Rydych yn cydnabod bod gennym ni'r hawl i atal neu derfynu'r grant a/neu fynnu eich bod yn ad-dalu'r grant cyfan neu ran ohono mewn unrhyw un o'r sefyllfaoedd a ganlyn. Mae'n rhaid i chi ein hysbysu os yw unrhyw un o'r sefyllfaoedd hyn wedi digwydd neu'n debygol o ddigwydd:`,
```
Comment (mattandrews) [inline controllers/apply/awards-for-all/terms.js]: 🤦‍♂

File: views/components/form-fields-next/macros.njk [hunk 4]
```diff
@@ -444,19 +444,20 @@
     {% call inputAddress(
         {
             name: field.name + '[previousAddress]',
-            label: 'Previous home address',
```
Comment (mattandrews) [inline views/components/form-fields-next/macros.njk]: Oops, I should've spotted this when making the big translation file... 🙈

**Pull request merged**